### PR TITLE
First draft of release notes for 1-6-16

### PR DESCRIPTION
**Pull request by srpearce**
This PR creates release notes for the Jan 6 edx Platform release.
Tagging product owners inline for the sections they should review; thanks!
## Doc task is [DOC-2528](https://openedx.atlassian.net/browse/DOC-2528)

Release page: https://openedx.atlassian.net/wiki/display/ENG/2016-01-05+Release
### Date Needed

Release is expected by 11:00am on Wednesday Jan 6
### Reviewers

Subject matter experts: 
- [ ] @explorerleslie 
- [ ] @marcotuts 
- [ ] @cptvitamin  
- [ ] @clrux 
- [ ] Doc team review (sanity check/copy edit/dev edit): @pdesjardins  

NOTE: Links might be broken until referenced doc is published.
### Testing
- [ ] Ran ./run_tests.sh without warnings or errors - except for not-yet-live links
### Post-review
- [ ] Squash commits

### CODE REVIEW

File: en_us/release_notes/source/2016/studio/studio_0106_2016.rst [hunk 1]
```diff
@@ -0,0 +1,17 @@
+
```
Comment (srpearce) [inline en_us/release_notes/source/2016/studio/studio_0106_2016.rst:1]: In the following, I'd like to remove the second sentence and just make sure this info is covered in the documentation. It's included here as a note to self.

File: en_us/release_notes/source/2016/documentation/doc_0106_2016.rst [hunk 2]
```diff
@@ -0,0 +1,37 @@
+==================================
+Documentation for edX Partners
+==================================
+
+Building and Running an edX Course
+************************************
+
+* Information about heading levels has been added to the
```
Comment (lamagnifica) [inline en_us/release_notes/source/2016/documentation/doc_0106_2016.rst]: I'd suggest "Guidance for the most appropriate heading levels to use in HTML components..."

File: en_us/release_notes/source/2016/documentation/doc_0106_2016.rst [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+
+==================================
+Documentation for Open edX Users
+==================================
+
+Building and Running an Open edX Course
+******************************************
+
+Information about heading levels has been added to the
```
Comment (lamagnifica) [inline en_us/release_notes/source/2016/documentation/doc_0106_2016.rst]: ditto

File: en_us/release_notes/source/2016/studio/studio_0106_2016.rst (hunk 1 above)
Comment (lamagnifica) [inline en_us/release_notes/source/2016/studio/studio_0106_2016.rst:1]: confirmed, I'll tune up my description now.

File: en_us/release_notes/source/2016/studio/studio_0106_2016.rst [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+* In the HTML component visual editor, the H1 and H2 options have been removed.
+  Because H1 and H2 headings correspond to pre-defined page elements, including
+  the web page title, additional H1 and H2 headings can interfere with the
+  functionality of tools such as screen readers. H4, H5, and H6 options have
+  been added. For more information, see :ref:`partnercoursestaff:` or
```
Comment (lamagnifica) [inline en_us/release_notes/source/2016/studio/studio_0106_2016.rst]: possibly phrase like, In addition to the H3 option, H4, H5, and H6 are now available!!!
(They are listed with Heading spelled out on the menu: Heading 3, but appear as < h3 > in the HTML fwiw.

File: en_us/release_notes/source/2016/openedx/openedx_0106_2016.rst [hunk 5]
```diff
@@ -0,0 +1,7 @@
+
+* When a course team uploads a very large file as a course image, Studio now
+  creates small and large thumbnails of the image for the learner dashboard and
+  the course About page. System administrators can configure the size of these
+  thumbnail images in Django.
```
Comment (pdesjardins) [inline en_us/release_notes/source/2016/openedx/openedx_0106_2016.rst]: Omit "in Django"? Or "configure the size of these thumbnail images in the Django administration interface"?

File: en_us/release_notes/source/2016/openedx/openedx_0106_2016.rst [hunk 6]
```diff
@@ -0,0 +1,7 @@
...
+
+* More?
```
Comment (pdesjardins) [inline en_us/release_notes/source/2016/openedx/openedx_0106_2016.rst]: Looks like this is waiting for someone to follow up. The changes I noted were all specific to edx.org.

File: en_us/release_notes/source/2016/studio/studio_0106_2016.rst [hunk 7]
```diff
@@ -0,0 +1,17 @@
...
+  :ref:`opencoursestaff:`.
+
+* Course teams can now specify additional rules and exceptions for proctored
+  exams. An **Advanced** tab has been added to the settings for the subsection
+  that contains the proctored exam. For more information, see
+  :ref:`partnercoursestaff:CA_ProctoredExams`.
```
Comment (pdesjardins) [inline en_us/release_notes/source/2016/studio/studio_0106_2016.rst]: (PHX-161) is the JIRA reference.

File: en_us/release_notes/source/2016/index.rst [hunk 8]
```diff
@@ -0,0 +1,13 @@
...
+edX Release Notes 2016
+####################################
+
+The following pages describe what is new in 2016.
+
+
+.. toctree::
+   :maxdepth: 1
+
+   01-06-2015
```
Comment (pdesjardins) [inline en_us/release_notes/source/2016/index.rst]: Should be 01-06-2016

File: en_us/release_notes/source/2016/openedx/openedx_0106_2016.rst (hunk 6 above)
Comment (srpearce) [inline en_us/release_notes/source/2016/openedx/openedx_0106_2016.rst]: Yeah, I'm going to take out the Open edX file.

File: en_us/release_notes/source/2016/website/website_0106_2016.rst [hunk 9]
```diff
@@ -0,0 +1,5 @@
+
+* The latest course in the edX Learning Series, :ref:`Creating Video for the
+  edX Platform`_, is now available.
+
+.. include:: ../../../links/links.rst
```
Comment (pdesjardins) [inline en_us/release_notes/source/2016/website/website_0106_2016.rst]: Go up one more directory I think. ../../../../links/links.rst

File: en_us/release_notes/source/2016/website/website_0106_2016.rst [hunk 10]
```diff
@@ -0,0 +1,5 @@
+
+* The latest course in the edX Learning Series, :ref:`Creating Video for the
```
Comment (pdesjardins) [inline en_us/release_notes/source/2016/website/website_0106_2016.rst]: Looks like the :ref: should be removed?

File: en_us/release_notes/source/2016/lms/lms_0106_2016.rst [hunk 11]
```diff
@@ -0,0 +1,25 @@
...
+* The video player now includes closed captions. Users can view closed captions
+  and transcripts either separately or at the same time. Currently, closed
+  captions and transcripts use the same file.
+
+* To increase accessibility, color contrast has been enhanced for some elements
+  of the Instructor Dashboard and the learner **Progress** page. (AC-177 and AC-178)
+
+* In the topic list on the course discussion page, a third indicator now lets
+  learners know that the user has read the topic, but that new comments have
+  been posted. (AC-295)
```
Comment (catong) [inline en_us/release_notes/source/2016/lms/lms_0106_2016.rst]: I think this is a bug fix, not addition of a new indicator @srpearce

**Comment by pdesjardins** [comment]
jenkins run continuous-integration/travis-ci/pr

**Comment by catong** [comment]
travis run continuous-integration/travis/ci/pr

**Pull request merged**